food.py

```diff
@@ -0,0 +1,97 @@
+import requests
+import os
+import re
+import time
+import ast
+
+def main():
+    dishes_found = {}
+    yelp_dish = ""
+
+    while dishes_found == {}:
+        dish_input = get_dish()
+        dishes_found = get_dish_info(dish_input)
+
+        if dishes_found == "error":
+            break
+
+        display_result(dishes_found)
+        if dishes_found != {}:
+            yelp_dish = search_restraunt(dishes_found)
+    return yelp_dish
+
+def get_dish():
+    while True:
+        dish_input = input('Enter dish name: ').strip().lower()
+
+        if not dish_input.replace(' ','').isalpha():
+            print("Input was either empty or contained an invalid character, please try again.")
+            continue
+        else:
+            dish_input = re.sub(' +', ' ', dish_input)
+            break
+    return dish_input
+
+def get_dish_info(dish_input):
+    response = request_dishes(dish_input)
+    
+    if response == "error":
+        return response
+    else:
+        dishes_found = convert_response(response)
+        return dishes_found
+
+def request_dishes(dish_input):
+    string = os.environ.get('headers')
```

Comment:
maybe do 
string = os.environ.get('headers', default='{}')
just in case the env doesn't get grabbed